lib/mime.ex

```diff
@@ -47,6 +48,7 @@ defmodule MIME do
     "application/vnd.openxmlformats-officedocument.wordprocessingml.document" => ["docx"],
     "application/vnd.rar" => ["rar"],
     "application/vnd.visio" => ["vsd"],
+    "application/wasm" => ["wasm"],
```

Comment:
Fun tidybit: we don't have `w` in portuguese, it is an "imported" letter, so in the portuguese alphabet, it comes after `x`. Hence my mistake! :D

Does the same happen in polish with `v`?

Reply:
Nope. Fortunately when someone encountered such problem they went with most reasonable solution - just use Latin order and add accented letters after their visual counterparts (so `ó` is after `o` not `u`). So `v` is between `u` and `w` and `x` is between `w` and `z`.